### ISSUE TITLE
Proposed detect-crop documentation addition

### ISSUE THREAD

**Issue by dennya** [lisamelton/video_transcoding#93]
To preview the crop that HandBrake suggested, select and copy the two lines that immediately follow the **# From HandBrakeCLI:** line and paste them into Terminal. A silent preview using that recommendation will run in an MPlayer window. To preview MPlayer's crop suggestion, repeat the process with the two lines following the **# From mplayer:** line to see the second suggested crop and make your determination. Select the appropriate **--crop** values for your transcode-video command.

**Comment by donmelton** [comment]
@dennya Thanks! I'll take a look at this next week when I'm working on the README again.

**Comment by donmelton** [comment]
@dennya OK, I'm finally looking at this again. And... I think your suggested text is too complicated. Sorry. Your text might actually be longer than the `detect-crop` output.

Here's what I have now:

> Just copy and paste the sample commands to preview or transcode.

I really think that's sufficient. If I add anything, I want it to be terse and to the point.

**Comment by dennya** [comment]
Fair enough. I take your point. I tend to over-explain in my documentation and leave terseness for screen output.

**Comment by donmelton** [comment]
@dennya OK, but I'm not closing the issue yet. Because I think we can still expand my original instructions without bloating them.

And you really have me thinking about this now. :) I'm just not sure yet what the right text should be.

Perhaps the expanded text should be added to this next line:

> You'll then need to preview both and decide which to use.

But maybe the sample output above is sufficient? Because what I don't want to do is just repeat the sample output.

This is a real head scratcher.

If it's any consolation, I'm much harder on my own text than yours. I just edited a multi-thousand-word addition to the "README" down to less than 750 words. And I'm not even sure about that yet. :)
